src/index.html

```diff
@@ -6,9 +6,76 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#"
+         class="logo"
+      >
+        <img src="images/logo.png"
+             alt="MoyoLogo"
+             class="logo__img"
+        >
+      </a>
```

Comment:
<img width="1069" alt="Attributes formatting" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/f21abd5f-37be-45d3-a0d7-5e9d9f79472b">

The same in other places